ios/browser/api/networking/utils/brave_certificate_ios_utils.mm

```diff
@@ -0,0 +1,398 @@
+//
+//  brave_certificate_ios_utils.mm
+//  CertificateViewer
+//
+//  Created by Brandon on 2021-05-18.
+//
```

Comment:
Incorrect file header

Reply:
at least put your last name! Lots of Brandon's out there 🙂